.circleci/config.yml

```diff
@@ -10,13 +10,22 @@ jobs:
     <<: *defaults
     steps:
       - run: sudo apt-get install -y libudev-dev
+      - run:
+          name: Install latest yarn
+          command: |
+            curl -fsSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" ;
```

Comment:
I don't get where `YARN_VERSION` is set 😅

Reply:
ahah, it's defined in CircleCI's environment

Reply:
🙇